runtime/autoload/health/provider.vim

```diff
@@ -148,7 +141,7 @@ endfunction
 function! s:check_python(version) abort
   call health#report_start('Python ' . a:version . ' provider')
 
-  let python_bin_name = 'python'.(a:version == 2 ? '2' : '3')
+  let python_bin_name = 'python'.(a:version == 2 ? '' : '3')
```

Comment:
Is this a good idea? python is not guaranteed to refer to python2. On modern systems, there's a PEP recommending that python2 executable (or link) should be provided, pointing to python version 2, however I don't know the adoption status.